### ISSUE TITLE
Request: Optional packages for LTSC 2019

### ISSUE THREAD

**Issue by supercilious** [DrEmpiricism/Optimize-Offline#14]
Since there is an option to integrate the Store and the Edge browser package, I would like to suggest that the following packages also be available:

| Package | Source | Contents |
|:-------------|:--------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Deduplication | Server 2019 | Microsoft-Windows-Dedup-ChunkLibrary-Package<br> Microsoft-Windows-Dedup-Package<br> Microsoft-Windows-FileServer-ServerCore-Package<br> Microsoft-Windows-VdsInterop-Package |
| Connect App | Windows 10 1809 (Non-LTSC) | PPIProjection ? |

**Comment by DrEmpiricism** [comment]
Ironically I was just extracting the Dedup Packages when I read this.  I will implement this once I make sure it does not negatively affect anything.

I will take a look at PPIProjection, too.

**Comment by supercilious** [comment]
For what it's worth, I can confirm that it works perfectly and does not negatively affect anything on my systems. I am running a LTSC 2019 x64 modified using this script and manually integrated using DISM the package CABs for dedup and PPIProjection.

I extracted the packages from Server 2019 for dedup and from Enterprise non-LTSC for PPIProjection using the "Aunty Mel's Cheap And Nasty SxS File Expander" scripts from MDL forums. Everything works fine, signatures are correct and there is no WinSxS corruption or any other issues. There has only been one update on WU since I did it, but it updated flawlessly of course.
